common/util_test.go

```diff
@@ -63,24 +63,16 @@ func TestCheckForDockerfile(t *testing.T) {
 
 func TestExtract(t *testing.T) {
```

Comment:
now that i think about it I feel this the `ExtractRepository` function should be in `git.go` 🤔

Reply:
I agree!